mkdocs_jupyter/convert.py

```diff
@@ -1,7 +1,9 @@
-import os
+import io
 import logging
+import os
 from copy import deepcopy
 
+import jupytext
```

Comment:
this feature would introduce a new dependency (used to convert the `.py` to a notebook that is, then, passed to your `nb2md` and `nb2html` functions via stringio. AFAIK, nbconvert only supports 1-way conversions from `.ipynb` to `.py`.